app/services/cclf/fee/mapping_fee_adapter.rb

```diff
@@ -32,6 +36,10 @@ def bill_key
         raise 'Implement in sub-class'
       end
 
+      def case_type_adapter
+        ::CCLF::CaseTypeAdapter.new(object.claim.case_type)
```

Comment:
💭 Probably worth _memoizing_ this method so that it doesn't initialise a new class every time is called.